packages/next/build/index.ts

```diff
@@ -208,10 +208,26 @@ export default async function build(dir: string, conf = null): Promise<void> {
     }),
   ])
 
+  const clientConfig = configs[0]
+
+  if (
+    clientConfig.optimization &&
+    (clientConfig.optimization.minimize == false ||
```

Comment:
Any reason this isn't strict? 

```suggestion
    (clientConfig.optimization.minimize !== true ||
```